tests/foreman/ui/test_org.py

```diff
@@ -821,26 +838,28 @@ def test_add_medium(self, medium_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        medium = entities.Media(
-            name=medium_name,
-            path_=INSTALL_MEDIUM_URL % gen_string('alpha', 6),
-            os_family='Redhat',
-        ).create()
-        self.assertEqual(medium.name, medium_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_medias=[medium_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_media'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % medium_name))
-            self.assertIsNotNone(element)
+            for medium_name in generate_strings_list():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.